mongodbatlas/resource_mongodbatlas_project.go

```diff
@@ -220,6 +232,10 @@ func resourceMongoDBAtlasProjectCreate(ctx context.Context, d *schema.ResourceDa
 
 	_, _, err = conn.Projects.UpdateProjectSettings(ctx, project.ID, projectSettings)
 	if err != nil {
+		_, errd := conn.Projects.Delete(ctx, project.ID)
+		if errd != nil {
+			return diag.Errorf(errorProjectDelete, project.ID, errd)
+		}
```

Comment:
Why can't we use `deleteProject()` here?

Reply:
Modified create to use this function in all delete project requests so that it uses function with timeout